cpp/ql/test/query-tests/Likely Bugs/Likely Typos/FutileParams/test.c

```diff
@@ -1,25 +1,25 @@
 void declared_empty();
 void declared_void(void);
 void declared_with(int);
-void declared_empty_defined_with();
```

Comment:
Why remove this declaration? The function name suggests that it's declared, but now it isn't.